ingressPort.tex

```diff
@@ -357,204 +361,122 @@ \section{Instruction Interface}
-    \begin{tabulary}{\textwidth}{|l|l|}
-        \hline
-        \textbf {Retired} & \textbf {Instruction Trace Block} \\
-        \hline
-        1000: \textbf{\textit{divuw}} &  \textbf{iretire}=7, \textbf{iaddr}=0x1000, \textbf{ntkn}=0, \textbf{itype}=8\\
-        1004: \textbf{\textit{add}} &  \\
-        1008: \textbf{\textit{or}} &  \\
-        100C: \textbf{\textit{c.jalr}} &  \\
-        \hline
-        0940: \textbf{\textit{addi}} &  \textbf{iretire}=4, \textbf{iaddr}=0x0940, \textbf{ntkn}=1, \textbf{itype}=5\\
-        0944: \textbf{\textit{c.beq}} &  \\
-        0946: \textbf{\textit{c.bnez}} &  \\
-        \hline
-        0988: \textbf{\textit{lbu}} &  \textbf{iretire}=4, \textbf{iaddr}=0x0988, \textbf{ntkn}=0, \textbf{itype}=0\\
-        098C: \textbf{\textit{csrrw}} &  \\
-        \hline
-    \end{tabulary}
-\end{table}
-
-
-\subsection {Sideband signals}
-
-In some circumstances there will be some sideband signals which may
-affect the encoder's behavior, for example to start and/or stop
-encoding.
-There will sometimes be cases where the encoder may be
-required to affect the behaviour of the core, for example stalling.
+Optional sideband signals may be included to provide additional functionality, as described in
+tables \ref{tab:ingress-side-band} and \ref{tab:egress-side-band}.
 
 Note, any user defined information that needs to be output by the encoder
-will need to be applied to the \textbf{context} value.
+will need to be applied via the \textbf{context} input.
 
 \begin{table}[htp]
     \centering
-    \caption{User Sideband Encoder Ingress signals}
+    \caption{Optional sideband encoder ingress signals}
     \label{tab:ingress-side-band}
-    \begin{tabulary}{\textwidth}{|l|l|}
+    \begin{tabulary}{\textwidth}{|l|p{100mm}|}
         \hline
         \textbf{Signal} & \textbf{Function} \\
-       \hline
-        \textbf{user} [\textit{user\_width\_p}-1:0] &  Filtering sideband signals (see Chapter~\ref{ch:filtering})\\
         \hline
-        \textbf{halted}& Core is stalled or halted \\
+        \textbf{impdef}[\textit{impdef\_width\_p}-1:0] &  Implementation defined sideband signals.  A typical use for
+        these would be for filtering (see Chapter~\ref{ch:filtering}.\\
+        \hline
+        \textbf{trigger}[1:0] & A pulse on bit 0 will cause the encoder to start tracing, and continue until further 
+        notice, subject to other filtering criteria also being met.\newline
+        A pulse on bit 1 will cause the encoder to stop tracing until further notice.  See section~\ref{sec:trigger}).\\
         \hline
-        \textbf{reset}& Core in reset \\
+        \textbf{halted}& Core is halted.  Upon assertion, the encoder will output a packet to report the address 
+        of the last instruction retired before halting, followed by a support packet to indicate that tracing has stopped. 
+        Upon deassertion, the encoder will start tracing again, commencing with a synchronisation packet.\\

```

Comment:
```suggestion
        Upon deassertion, the encoder will start tracing again, commencing with a synchronization packet.\\
```
I got the impression from changes you made to the synchronization section in branchTrace.tex that you're trying to change to American spellings.  (I'm OK either way but it should be consistent.)